styles/main.css

```diff
@@ -19,7 +22,7 @@ html {
 }
 
 body {
-  font-family: helvetica, sans-serif;
+  font-family: bebas-neue, helvetica, sans-serif;
   /*adjust font size to calibrate for rem*/
```

Comment:
we need to move this comment to the html selector with the font-size rem conversion.
merging now since its not a big issue though.